Resources/Prototypes/Entities/Objects/Weapons/Bombs/hot_potato.yml

```diff
@@ -0,0 +1,91 @@
+- type: entity
+  name: hot potato
+  description: A label on the side of this potato reads "Product of Donk Co. Service Wing. Activate far away from populated areas. Device will only attach to sapient creatures."
+  parent: BaseItem
+  id: HotPotato
```

Comment:
just make HotPotato parent from HotPotatoFake and add in the explosive components. They share about 80% of the yaml anyways.